From 2b76afbf1fdebf2309be35d9980571316ee3f3a8 Mon Sep 17 00:00:00 2001
From: John Greene <jogreene@redhat.com>
Date: Mon, 8 Sep 2014 15:07:18 -0400
Subject: [ethernet] ixgbevf: Miscellaneous conversions to ETH_ALEN

Message-id: <1410188883-6595-4-git-send-email-jogreene@redhat.com>
Patchwork-id: 92350
O-Subject: [RHEL7.1 PATCH 03/48] [ixgbevf] net:drivers/net: Miscellaneous conversions to ETH_ALEN
Bugzilla: 1091124
RH-Acked-by: Neil Horman <nhorman@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>
RH-Acked-by: John Linville <linville@redhat.com>

BZ: 1091124
Brew: https://brewweb.devel.redhat.com/taskinfo?taskID=7919168
Tested: In beaker. See BZ for details

[redhat] only for ixgbevf

upstream: commit d458cdf712e0c671e8e819abb16ecd6e44f9daec

Convert the memset/memcpy uses of 6 to ETH_ALEN where appropriate.
Also convert some struct definitions and u8 array declarations of [6] to
ETH_ALEN.
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/intel/ixgbevf/vf.c b/drivers/net/ethernet/intel/ixgbevf/vf.c
index 387b526..4d44d64 100644
--- a/drivers/net/ethernet/intel/ixgbevf/vf.c
+++ b/drivers/net/ethernet/intel/ixgbevf/vf.c
@@ -242,7 +242,7 @@ static s32 ixgbevf_set_uc_addr_vf(struct ixgbe_hw *hw, u32 index, u8 *addr)
  msgbuf[0] |= index << IXGBE_VT_MSGINFO_SHIFT;
  msgbuf[0] |= IXGBE_VF_SET_MACVLAN;
  if (addr)
-  memcpy(msg_addr, addr, 6);
+  memcpy(msg_addr, addr, ETH_ALEN);
  ret_val = mbx->ops.write_posted(hw, msgbuf, 3);
 
  if (!ret_val)
@@ -275,7 +275,7 @@ static s32 ixgbevf_set_rar_vf(struct ixgbe_hw *hw, u32 index, u8 *addr,
 
  memset(msgbuf, 0, sizeof(msgbuf));
  msgbuf[0] = IXGBE_VF_SET_MAC_ADDR;
- memcpy(msg_addr, addr, 6);
+ memcpy(msg_addr, addr, ETH_ALEN);
  ret_val = mbx->ops.write_posted(hw, msgbuf, 3);
 
  if (!ret_val)
-- 
1.7.1